src/index.html

```diff
@@ -1,13 +1,64 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" >
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./styles/index.css" >
+  </head>
+  <body class="card">
+    <div class="card__container" data-qa="card">
+      <img 
+        class="card__img" 
+        src="./images/imac.jpeg"
+        alt="apple imac">
+      <a class="card__name" href="#">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </a>
+      <p class="card__code">
+        Product code: 195434
+      </p>
+      <div class="card__stars stars">
+        <div class="stars__block">
+          <div class="stars__img"></div>
+
```

Comment:
You don't need to put an empty line between one-line sibling tags. But put it between multiline tags. Fix it in the whole code.